api/src/main/java/org/openmrs/module/eptsreports/metadata/Metadata.java

```diff
@@ -175,33 +175,33 @@ public static Concept getConcept(String lookup) {
 	
 	/**
 	 * @return the List of Concepts that matches the passed comma-separated list of concept lookups
-	 * @see MetadataLookup#getConcept(String)
+	 * @see Metadata#getConcept(String)
 	 */
 	public static List<Concept> getConceptList(String lookup) {
```

Comment:
This is lower priority at the moment and we can add this to the work plan for later, but out of interest are there any unit tests for these functions?

Reply:
I actually borrowed this class from [pihmalawi module](https://github.com/PIH/openmrs-module-pihmalawi/blob/master/api/src/main/java/org/openmrs/module/pihmalawi/metadata/Metadata.java).

The class is extended by [CommonMetadata](https://github.com/esaude/openmrs-module-eptsreports/blob/MEPTS-24/api/src/main/java/org/openmrs/module/eptsreports/metadata/CommonMetadata.java)  and because all the specific metadata classes (...HivMetadata, TbMetada, ...) extends this CommonMetadata class they all get access to the metadata lookup methods from the Metadata class. -I haven't seen any unit tests for this methods tho I think we should include a XML dataset file and add the unit tests.

Reply:
Ok cool, I'll add a ticket to our backlog to add some tests when we have some